packages/react-ui/components/Modal/__stories__/Modal.stories.tsx

```diff
@@ -767,3 +751,31 @@ export const AlignCenterAndNoClose = () => (
   </Modal>
 );
 AlignCenterAndNoClose.story = { parameters: { creevey: { captureElement: null } } };
+
+const Header = () => <Modal.Header>Header</Modal.Header>;
+const Body = () => (
+  <Modal.Body>
+    {new Array(200).fill('Use rxjs operators with react hooks.').map((item, index) => (
+      <p key={index}>{item}</p>
+    ))}
+  </Modal.Body>
+)
+const Footer = () => <Modal.Footer>Footer</Modal.Footer>;
+
+const ModalWithChildrenFromOtherComponent = (sticky = true) => {
+  return (
+    <Modal>
+      <Header/>
+      <Body />
+      <Footer />
+    </Modal>
+  );
+}
+
+export const ModalWithChildrenFromOtherComponentWithStickyElements: CSFStory<JSX.Element> = () => (
+  ModalWithChildrenFromOtherComponent()
+)
+
+ModalWithChildrenFromOtherComponentWithStickyElements.story = {
+  parameters: { creevey: { tests: TopMiddleBottomModalTests } }
+};
```

Comment:
Тут можно убрать лишнюю обёртку и сократить название т.к. `sticky` по умолчанию `true`.

```suggestion
export const ModalWithChildrenFromOtherComponent = () => (
  <Modal>
    <Header />
    <Body />
    <Footer />
  </Modal>
);

ModalWithChildrenFromOtherComponent.story = {
  parameters: { creevey: { tests: TopMiddleBottomModalTests } },
};
```